server/.dockerignore

```
@@ -1,3 +1,4 @@
 env
 .dockerignore
-Dockerfile
\ No newline at end of file
+Dockerfile
+venv
```

Should add trailing newline.
```suggestion
venv

```